### PR TITLE
Feature/streamline-startup

### PR DESCRIPTION
**Pull request by DillonOLeary**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description

This PR just changes the startup documentation to make it easier to understand. In helping other developers get set up I realized that there was room for improvement.

View new Readme here: https://github.com/digitalaidseattle/seattle-humane-app/blob/feature/streamline-startup/README.md

## Related Tickets & Documents

None

## QA Instructions, Screenshots, Recordings

I deleted the application and went through the instructions a few times to make sure they were clear.

## Added/updated tests?
_We are aiming for 100% statement coverage._

- [ ] Yes
- [x ] No, and this is why: Documentation change
- [ ] I need help with writing tests

## Are there any post-deployment tasks we need to perform?
I will need another PR that allows devs to bypass OAuth when developing locally. It's a pain without that.

## What gif best describes this PR or how it makes you feel?

starting car

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,81 +28,110 @@ The website is built using **Next.js**, a React-based framework for server-side
...
    git clone https://github.com/openseattle/seattle-humane-app
-   cd seattle-humane-app
-   git checkout main
-   npm install
    ```
 
-2. Setup VSCode for auto-formatting
+1. Open VS code.
+
+1. From VS code, open the command palette using `⇧⌘P` and type:
```
Comment (sharrief) [inline README.md:56]: Also include the Windows shortcut ctrl+shift+P?

File: README.md [hunk 2]
```diff
@@ -28,81 +28,110 @@ The website is built using **Next.js**, a React-based framework for server-side
...
-Local requires a few one-time steps. You will need to create a Google OAuth ID, put information about it into your .env.local file, start Supabase, put values from Supabase into your .env.local file, and then finally start the web app.
+Local setup requires a few one-time steps. In the following steps, you will start Supabase locally and put values from Supabase into your .env.local file.
 
-1. Create a copy of .env.local.example
+1. Create a copy of .env.local.example:
 
    ```bash
    cp .env.local.example .env.local
    ```
+
```
Comment (sharrief) [inline README.md:76]: You also need to call 'source .env.local' right?

Comment (sharrief) [reply README.md:76]: Oops nvm I see it further below

**Comment by DillonOLeary** [comment]
Probably easiest to view the Readme changes here: https://github.com/digitalaidseattle/seattle-humane-app/blob/feature/streamline-startup/README.md

**Review by sharrief — APPROVED** [review]
Thanks! Looks very thorough now without being too long :)

File: README.md (hunk 1 above)
Comment (DillonOLeary) [reply README.md:56]: Good call! I'll create a new PR with this

**Pull request merged**